vizro-core/examples/dev/app.py

```diff
@@ -384,6 +516,73 @@
 
 tabs = vm.Page(title="Tabs", components=[vm.Tabs(tabs=[tab_1, tab_2])], controls=[vm.Filter(column="continent")])
 
+
+tooltip = vm.Page(
+    title="Tooltip",
+    layout=vm.Grid(grid=[[0], [0], [1], [1], [1], [1], [1], [1], [2]]),
+    components=[
+        vm.Card(
+            text="""
+                The `description` argument enables you to add helpful context to your components by displaying an
+                info icon next to its title. Hovering over the icon reveals a tooltip with the text you provide.
+
+                Tooltips can be added to any Vizro component that has a `title` argument.
+                You can provide a string to use the default info icon or `Tooltip` model to use any icon from the
+                [Google Material Icons library](https://fonts.google.com/icons).
+                Tooltips provide clean and lightweight way to add additional details to your dashboard.
```

Comment:
Formatting is off on my suggestion, but I would slightly rephrase to make this clearer
```suggestion
The description argument allows you to add helpful context to your components by displaying a small info icon next to the component's title. When users hover over the icon, a tooltip appears showing the text you provide.

You can add tooltips to any Vizro component that supports the title argument. The description accepts:

* A string, which uses the default info icon.
* A Tooltip model, which lets you customize the icon using any symbol from the [Google Material Icons library](https://fonts.google.com/icons)

Tooltips are a clean, lightweight way to offer additional details without cluttering your dashboard.
```